### PR TITLE
x11-misc/xflux-gui: New package

### PR DESCRIPTION
**Pull request by ConiKost**
x11-misc/xflux-gui: New package

Depends on: https://bugs.gentoo.org/606128

Closes: https://bugs.gentoo.org/606130

### CODE REVIEW

File: x11-misc/xflux-gui/xflux-gui-1.1.10.ebuild [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+DESCRIPTION="A GUI for f.lux"
+HOMEPAGE="https://justgetflux.com/"
+SRC_URI="https://github.com/xflux-gui/fluxgui/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="-* ~amd64 ~x86"
+LICENSE="MIT"
+SLOT="0"
+
+RDEPEND="dev-libs/libappindicator:2[python]
```
Comment (mgorny) [inline x11-misc/xflux-gui/xflux-gui-1.1.10.ebuild]: All Python packages always need `[${PYTHON_USEDEP}]` to ensure that matching Python implementation is enabled.

File: x11-misc/xflux-gui/xflux-gui-1.1.10.ebuild [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+	dev-python/gconf-python
+	x11-misc/xflux"
+
+S="${WORKDIR}/fluxgui-${PV}"
+
+pkg_postinst() {
```
Comment (mgorny) [inline x11-misc/xflux-gui/xflux-gui-1.1.10.ebuild]: `pkg_postrm` too.

File: x11-misc/xflux-gui/xflux-gui-1.1.10.ebuild [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+PYTHON_COMPAT=( python2_7 )
+
+inherit distutils-r1 gnome2-utils
+
+DESCRIPTION="A GUI for f.lux"
+HOMEPAGE="https://justgetflux.com/"
+SRC_URI="https://github.com/xflux-gui/fluxgui/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="-* ~amd64 ~x86"
```
Comment (mgorny) [inline x11-misc/xflux-gui/xflux-gui-1.1.10.ebuild]: Is the `-*` strictly necessary on this package?

**Comment by mgorny** [comment]
Oh, and please rebase against master to please CI.

File: x11-misc/xflux-gui/xflux-gui-1.1.10.ebuild (hunk 3 above)
Comment (ConiKost) [reply x11-misc/xflux-gui/xflux-gui-1.1.10.ebuild]: Depends. As it needs x11-misc/xflux, which is only a binary package for amd64 and x86, it doesn't make sense on others arches?

File: x11-misc/xflux-gui/xflux-gui-1.1.10.ebuild (hunk 1 above)
Comment (ConiKost) [reply x11-misc/xflux-gui/xflux-gui-1.1.10.ebuild]: I've added ${PYTHON_USEDEP}.

File: x11-misc/xflux-gui/xflux-gui-1.1.10.ebuild (hunk 2 above)
Comment (ConiKost) [reply x11-misc/xflux-gui/xflux-gui-1.1.10.ebuild]: Added also.

**Comment by ConiKost** [comment]
Should be now rebased against master.

File: x11-misc/xflux-gui/xflux-gui-1.1.10.ebuild [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+LICENSE="MIT"
+SLOT="0"
+
+RDEPEND="dev-libs/libappindicator:2[python,${PYTHON_USEDEP}]
+	dev-python/gconf-python[${PYTHON_USEDEP}]
+	dev-python/pexpect[${PYTHON_USEDEP}]
+	dev-python/pygtk[${PYTHON_USEDEP}]
+	dev-python/pyxdg[${PYTHON_USEDEP}]
+	gnome-base/libglade[${PYTHON_USEDEP}]
```
Comment (mgorny) [inline x11-misc/xflux-gui/xflux-gui-1.1.10.ebuild]: You went one package too far. Here PYTHON_TARGETS are only used for USE=tools.

Comment (mgorny) [reply x11-misc/xflux-gui/xflux-gui-1.1.10.ebuild]: (but I'll fix that, no point in you remaking the PR for it)

Comment (ConiKost) [reply x11-misc/xflux-gui/xflux-gui-1.1.10.ebuild]: Whoops, sorry, next time, I will look through the ebuild deps, when looking through python deps.

**Comment by mgorny** [comment]
Thanks!

**Pull request closed without merging**